Content.Shared/SharedGameTicker.cs

```diff
@@ -114,5 +116,72 @@ public override void WriteToBuffer(NetOutgoingMessage buffer)
                 buffer.Write(TextBlob);
             }
         }
+        public struct RoundEndPlayerInfo
+        {
+            public string PlayerOOCName;
+            public string PlayerICName;
+            public string Role;
+            public bool Antag;
+
+        }
+
+        protected class MsgRoundEndMessage : NetMessage
+        {
+
+            #region REQUIRED
+
+            public const MsgGroups GROUP = MsgGroups.Command;
+            public const string NAME = nameof(MsgRoundEndMessage);
+            public MsgRoundEndMessage(INetChannel channel) : base(NAME, GROUP) { }
+
+            #endregion
+
+            public string GamemodeTitle;
+            //TODO: Change to a more detailed measurement of time.
+            public uint DurationInHours;
```

Comment:
It'd be best if you used TimeSpan for this, most rounds do not last much longer than an hour.

Reply:
Yeah, this was just a temp thing.